test/app.test.js

```diff
@@ -367,14 +367,25 @@ describe("app", function() {
                     })
                     .run();
             });
+              it('should not accept SA ID numbers with invalid dates', function () {
+                return tester
+                    .setup.user.state('SA_ID_Entry')
+                    .input('9502294800087')
+                    .check.interaction({
+                        state: 'ID_Type_Entry',
+                        reply: /The format of the ID number was incorrect/
+                    })
+                    .run();
+            });
+
 
             it('should accept valid SA ID numbers', function () {
                 return tester
                     .setup.user.state('SA_ID_Entry')
                     .setup.user.answers({
                         'Facility_Code_Entry': '111111'
                     })
-                    .input('8905100273087')
+                    .input('9602294800085')
```

Comment:
What's the significance of changing this? If it serves no purpose, you can just leave it as is.